core/platform/email/gae_email_services.py

```diff
@@ -63,13 +84,14 @@ def send_mail(
     if not feconf.CAN_SEND_EMAILS:
         raise Exception('This app cannot send emails.')
 
-    if not mail.is_email_valid(sender_email):
```

Comment:
Can we also add this check into our own validator function? (Maybe in future if gae adds or changes their validator function?)

Reply:
I will start working on this!
Thanks!

Reply:
Done